### DISCUSSION TITLE
Please set npm's page background

### DISCUSSION THREAD

**Discussion by RichardJECooke** [npm/feedback#214] [Website] [answered]
If you're using a browser with a dark theme then it shows that the npm programmers didn't even bother to set the page color, making the site unusable:

![image](https://user-images.githubusercontent.com/5468805/107612202-656fb280-6c4e-11eb-850e-85ad74941596.png)

**Comment by MylesBorins** [comment]
hey @RichardJECooke. how are you setting the dark theme? What browser and operating system?

I've tried dark mode on OSX with both chrome + Safari as well as Dark Theme on Android with Chrome and have been unable to recreate your report

**Comment by RichardJECooke** [comment]
Firefox - about:config - browser.display.background_color - #000000

Sets a new tab to black so it doesn't blind you at night. Most sites set the page background whatever color they use, usually white. But npm doesn't.

**Reply by MylesBorins** [reply]
Thanks for clarifying... I've opened a ticket with our engineering team

**Reply by CtrlAltCuteness** [reply]
As an FYI / extra "forwarding info with context", this also applies to embedded web browsers that are affixed and also support alpha color background transparency as shown in this [very large] image.

![image](https://user-images.githubusercontent.com/8661411/114744063-25868380-9d02-11eb-8b8a-b15bd5670400.png)

Context: _Tabletop Simulator_ on Steam

Hopefully final Edit: I'd assume the same would happen if I used my phone to make an overlay in Tasker's built-in Scene or the AutoApps app AutoTools Web Screen thing.

**Reply by MylesBorins** [reply]
@CtrlAltCuteness thanks for sharing! This is on the engineering backlog to get fixed but unsure what the timeline is. I'm checking with the team

**Comment by MylesBorins** [comment, marked as answer]
This should now be fixed 🎉

**Reply by CtrlAltCuteness** [reply]
Unless Tabletop Simulator changed, I can confirm it works now.

**Why does it happen?**

```css
html, body {
    /* Acts kind-of like so: */
    background-color: rgba(0, 0, 0, 0.0);
    color: rgba(0, 0, 0, 1.0);
}
```
